ignitionrobotics/ign-gazebo
# Upgrade ign-sensors and support custom sensors

Part of https://github.com/ignitionrobotics/ign-sensors/issues/18

## Summary


Requires https://github.com/ignitionrobotics/ign-plugin/pull/35 and https://github.com/ignitionrobotics/ign-sensors/pull/90.

This is an alternative to #304.

tl;dr: Loading symbols globally is a way of making we find them after they've been registered by another library.

---

TODO: write up a summary of what I think is happening.

## Test it

* Use all the PRs listed above
* Run worlds that contain sensors, like `ign gazebo sensors.sdf` or `ign gazebo sensors_demo.sdf`
* Sensor data should be available through topics, camera images should be visible on widgets

## Checklist
- [x] Added example world and/or tutorial
- [x] While waiting for a review on your PR, please help review
[another open pull request](https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3Aignitionrobotics+archived%3Afalse+)
to support the maintainers

**Note to maintainers**: Remember to use **Squash-Merge**
